exercises/minesweeper/example.js

```diff
@@ -0,0 +1,64 @@
+const BOMB = "*";
+
+class Minesweeper {
+  constructor() {
+    this.deltasToAdjacentSquares = [
+      [-1, -1],
+      [-1, 0],
+      [-1, 1],
+
+      [1, 1],
+      [1, 0],
+      [1, -1],
+
+      [0, 1],
+      [0, -1]
+    ]
+  }
+
+  annotate(rows) {
+    if (noDataPresent(rows)) {
+      return rows;
+    }
+
+    const inputBoard = rows.map(row => [...row]);
+    const outputBoard = [];
+    for (let x = 0; x < inputBoard.length; x++) {
```

Comment:
I would expect `Array.forEach` instead of two nested `for` loops. Is there a reason not to use `Array.forEach`? 

I'm not sure if it can provide you the index of the iteration. They seem to be needed to pass them to `countAdjacentBombs`.